### ISSUE TITLE
Santroller HID strum issue

### ISSUE THREAD

**Issue by undeadrattler** [YARC-Official/YARG#918]
Using the latest version of Santroller and the latest version of YARG, as well as testing with YARG 0.12.2 and 0.12.4 to make sure I'm not running into issues from ticket #871 - I have run into a debilitating issue where, due to the strum being reported to the kernel as an axis, I am unable to map it in-game. All other buttons and analog axes work fine (whammy, directional hat, tone select) and map without issue. I've attempted holding the strum down to zero the axis before trying to register it for an up strum, and vice-versa for a down strum; no dice. I'd rather not have to open my guitars again to solder the strum to the directional hat, and with everything testing completely fine in jstest and working great in windows, I can only assume it's something in how the axes are handled in YARG.

Let me know if you need any other information and I'll be happy to assist.

**Comment by ornelasf1** [comment]
I faced something similar to this. The nightly build of YARG (version b2879) got it working for me 👍 . The controller Im using is an HID device, with firmware by Santroller on a PI Pico

**Comment by theli-ua** [comment]
yeah, I couldn't make it work in linux on stable.  this works perfectly on the nightly though
